From 222c98c7979084fbefb4ce2ae377210c6e42011e Mon Sep 17 00:00:00 2001
Message-Id: <222c98c7979084fbefb4ce2ae377210c6e42011e.1650796268.git.geliang.tang@suse.com>
From: huangxuesen <huangxuesen@kuaishou.com>
Date: Mon, 6 Dec 2021 09:47:16 +0800
Subject: [PATCH] libbpf: Fix trivial typo
Git-commit: 222c98c7979084fbefb4ce2ae377210c6e42011e
Patch-mainline: v5.17-rc1
References: bsn#127
Modified-by-SEL: No


Fix typo in comment from 'bpf_skeleton_map' to 'bpf_map_skeleton'
and from 'bpf_skeleton_prog' to 'bpf_prog_skeleton'.

Signed-off-by: huangxuesen <huangxuesen@kuaishou.com>
Signed-off-by: Andrii Nakryiko <andrii@kernel.org>
Link: https://lore.kernel.org/bpf/1638755236-3851199-1-git-send-email-hxseverything@gmail.com
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 tools/lib/bpf/libbpf.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/lib/bpf/libbpf.h b/tools/lib/bpf/libbpf.h
index c0d62dd37c5d..2fa046a96142 100644
--- a/tools/lib/bpf/libbpf.h
+++ b/tools/lib/bpf/libbpf.h
@@ -1038,11 +1038,11 @@ struct bpf_object_skeleton {
 	struct bpf_object **obj;
 
 	int map_cnt;
-	int map_skel_sz; /* sizeof(struct bpf_skeleton_map) */
+	int map_skel_sz; /* sizeof(struct bpf_map_skeleton) */
 	struct bpf_map_skeleton *maps;
 
 	int prog_cnt;
-	int prog_skel_sz; /* sizeof(struct bpf_skeleton_prog) */
+	int prog_skel_sz; /* sizeof(struct bpf_prog_skeleton) */
 	struct bpf_prog_skeleton *progs;
 };
 
-- 
2.26.2

